src/vba/psychrolib.bas

```diff
@@ -635,21 +695,22 @@ Function GetTWetBulbFromHumRatio(ByVal TDryBulb As Variant, ByVal HumRatio As Va
 
   ' Bisection loop
   Tol = GetTol()
-  While (TWetBulbSup - TWetBulbInf > Tol)
+  index = 0
+  While (((TWetBulbSup - TWetBulbInf) > Tol) Or (index < MAX_ITER_COUNT))
```

Comment:
The test on index < MAX_ITER_COUNT is wrong. It should be a separate line of code inside the loop (same as in other languages) and if not satisfied should send to ErrHandler